core/trino-main/src/main/java/io/trino/operator/project/InputChannels.java

```diff
@@ -25,15 +26,23 @@
 public class InputChannels
 {
     private final int[] inputChannels;
+    private final int[] eagerlyLoadedChannels;
 
     public InputChannels(int... inputChannels)
     {
         this.inputChannels = inputChannels.clone();
+        this.eagerlyLoadedChannels = new int[0];
     }
 
     public InputChannels(List<Integer> inputChannels)
```

Comment:
is this constructor sill used?

Reply:
There are still a couple of other usages